Code/GraphMol/MolDraw2D/DrawTextSVG.cpp

```diff
@@ -12,13 +12,20 @@
 
 #include <sstream>
 #include <boost/algorithm/string.hpp>
+#include <boost/format.hpp>
 
 #include <GraphMol/MolDraw2D/DrawTextSVG.h>
 #include <GraphMol/MolDraw2D/MolDraw2DSVG.h>
 #include <GraphMol/MolDraw2D/MolDraw2DDetails.h>
 
 namespace RDKit {
 
+namespace {
```

Comment:
this should be deleted after the function is moved

Reply:
This still needs to be cleaned up

Reply:
Sorry, missed that one.